MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include *.txt
 include tests/*.py
 include tox.ini
 include vulture/whitelists/*.py
+include vulture/lines.py
```

Comment:
Are you sure this is necessary? The file should be covered by the vulture module in setup.py, right?